accounts/scwallet/hub.go

```diff
@@ -153,7 +153,7 @@ func (hub *Hub) setPairing(wallet *Wallet, pairing *smartcardPairing) error {
 
 // NewHub creates a new hardware wallet manager for smartcards.
 func NewHub(daemonPath string, scheme string, datadir string) (*Hub, error) {
-	context, err := pcsc.EstablishContext(daemonPath, pcsc.ScopeSystem)
+	context, err := pcsc.EstablishContext(pcsc.ScopeSystem)
```

Comment:
You went back to an older version of pcsc, this will break features.

Reply:
I did it by accident, now it's fixed.